src/server/page_server.cpp

```diff
@@ -399,10 +437,22 @@ page_server::set_passive_tran_server_connection (cubcomm::channel &&chn)
   assert (is_page_server ());
 
   chn.set_channel_name ("PTS_PS_comm");
+
+  const auto channel_id = chn.get_channel_id ();
+
   er_log_debug (ARG_FILE_LINE, "Passive transaction server connected to this page server. Channel id: %s.\n",
-		chn.get_channel_id ().c_str ());
+		channel_id.c_str ());
 
   m_passive_tran_server_conn.emplace_back (new connection_handler (chn, transaction_server_type::PASSIVE, *this));
+
+  {
+    std::lock_guard<std::mutex> lockg { m_pts_oldest_active_mvccids_mtx };
+    /* The entry is already created when ths PTS is connected. */
```

Comment:
```suggestion
    /* The entry must not already be present. If the same passive transaction server has been connected
     * before, the entry must have been removed when the PTS disconnected or when the connection
     *  to the PTS was aborted.
     */
```

Reply:
Oh, my bad. This is a Copy&Paste mistake. Thank you.